lib/syntax_tree/node.rb

```diff
@@ -6322,8 +6233,10 @@ def format_break(q, force:)
         q.format(node.consequent)
       end
 
-      force ? q.breakable_force : q.breakable_space
-      q.text("end")
+      unless keyword == "elsif"
```

Comment:
I added this conditional here because an `elsif` is always a consequent, so it should never be adding the `end` keyword to the end of the conditional block. Let me know if there's a better way to do this!